core/new-gui/src/app/common/util/url.ts

```diff
@@ -11,3 +12,10 @@ export function getWebsocketUrl(endpoint: string): string {
   websocketUrl.protocol = websocketUrl.protocol.replace("http", "ws");
   return websocketUrl.toString();
 }
+/**
+ * Generate a URL based on hostname, port and path.
+ */
+export function createUrl(hostname: string, port: number, path: string): string {
+  const protocol = location.protocol === "https:" ? "wss" : "ws";
+  return normalizeUrl(`${protocol}://${hostname}:${port}${path}`);
+}
```

Comment:
we have similar helper function called `getWebsocketUrl` in `src/app/common/util/url.ts`. Can we use that one?

Reply:
It turns out we don't need this one as well, so removed all the changes in this file.